bin/cron/export_mysql_database_to_redshift

```diff
@@ -21,13 +20,6 @@ NOTIFICATION_CHANNEL = 'data-alerts'
 
 def main
   ChatClient.message NOTIFICATION_CHANNEL, 'Beginning daily export from Aurora MySQL database to Redshift.'
-  # Delete clone of production cluster if it was incorrectly left behind by a previous execution of the import process.
-  Cdo::RDS.delete_cluster(RedshiftImport::CLONE_CLUSTER_ID)
-  Cdo::RDS.clone_cluster(
-    source_cluster_id: CDO.db_cluster_id,
-    clone_cluster_id: RedshiftImport::CLONE_CLUSTER_ID,
-    instance_type: RedshiftImport::CLONE_DB_INSTANCE_ID
```

Comment:
Can these constants also be removed from https://github.com/code-dot-org/code-dot-org/blob/94020ea309cfd32d189c686f5cd73065f117a6e5/lib/cdo/redshift_import.rb#L4-L5 ?